### ISSUE TITLE
Pulling John Bill's cigarette while John is on the same tile as the cigarette pulls John instead

### ISSUE THREAD

**Issue by Ormakent** [goonstation/goonstation#16386]
### Labels

[BUG]

### Description

title

### Steps to reproduce

1. Remove John's cigarette
2. Try to pull it
3. Pull John instead 

### Expected Behavior

would like to pull the object i'm trying to pull

### Additional Information & Screenshots

Cig also missing context menu sprite it seems
Bad quality vid example 

https://github.com/goonstation/goonstation/assets/35436986/f2aca3cc-2948-40bf-9fc6-dabcc77e898c

**Comment by MeggalBozale** [comment]
Yeah somewhere in the codebase it adds a \ that escapes it so it thinks u wanna pull John Bill this is probably not going to get fixed for a while

**Comment by amylizzle** [comment]
#1780

**Comment by TacoGS** [comment]
Yeah, this has to do more with the name itself in the warcrimes.dm not so much the fix 24779. I will be deploying a new PR for this one in a bit.
